src/components/_helpers/Wrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// I N T E R F A C E
+export interface WrapperProps {
+  height?: string
+  bgColor?: WrapperBgColor
```

Comment:
Why is this component in a `_helpers` folder? Is it use only for the documentation? If so I guess it make sense to not put it directly under `src`.

Reply:
Will be replaced by another following ticket